tuf/formats.py

```diff
@@ -403,8 +403,8 @@
 MIRROR_SCHEMA = SCHEMA.Object(
   object_name = 'MIRROR_SCHEMA',
   url_prefix = securesystemslib.formats.URL_SCHEMA,
-  metadata_path = RELPATH_SCHEMA,
-  targets_path = RELPATH_SCHEMA,
+  metadata_path = SCHEMA.Optional(RELPATH_SCHEMA),
+  targets_path = SCHEMA.Optional(RELPATH_SCHEMA),
```

Comment:
Prior to this change a `check_match()` call will ensure that `both metadata_path` and `targets_path` are set. But following this change it's possible to pass an object conformant to `MIRROR_SCHEMA` with neither value set.

I'm not sure whether we should worry too much about this, it should be fairly obvious to a client if they misconfigure by not setting either value?

Reply:
This is true, does not seem like a major issue to me either.